cmd/mt-fakemetrics/cmd/datafeed.go

```diff
@@ -101,6 +104,9 @@ times %4d orgs: each %s, flushing %d metrics so rate of %d Hz. (%d total unique
 				// we already sent, so we must increase the timestamp
 				if m == 0 {
 					ts += mp
+					if ts >= now && stopAtNow {
+						break METRICS
+					}
```

Comment:
there's another problem here I think which is that we never break out of the tick loop, so fakemetrics would keep running and ticking and possibly re-producing some metrics. in my pr i'll break out of the tick loop also

Reply:
ah never mind that, I think this will be caught by the `if ts >= now && stopAtNow {` condition further down